### DISCUSSION TITLE
ScriptReference/Vector3.Reflect

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#298] [Notes]
# ScriptReference/Vector3.Reflect

https://docs.unity3d.com/ScriptReference/Vector3.Reflect.html

**Comment by joonjoonjoon** [comment]
The image on this page is very misleading.  
If you reflect a position instead of a direciton, it would end up on the "other side" of the mirror plane (bottom of the picture), instead of to the side like the picture currently shows.

**Comment by vertxxyz** [comment]
This function expects the normal to be normalised.
If it's not, then the returned value will not be the same magnitude and the reflection will face in an unwanted direction.
